JS/ViewControllers/libraryChooseMediaLanguageViewController.js

```diff
@@ -0,0 +1,118 @@
+class LibraryChooseMediaLanguageViewController extends ViewController{
+
+    constructor() {
+        super();
+        this.shortLanguage;
+        this.language;
+        this.availableLanguages;
+    }
+
+    renderHtml(html) {
+        const htmlView = `
+            <section id="LibraryChooseMediaLanguageViewController" class="container">
+                <div class="row row-100 m12">
+                    <div class="col s12">
+                        <h1 class="center">Choose media language</h1>
+                    </div>
+                </div>
+                <div class="row row-25 m12">
+                    <form action="#">
+                        <select id="select-language" class="browser-default">
+                            <option value="none" selected></option>
+                        </select>
+                    </form>
+                </div>
+                <p id="demo"></p>
+                <div class="row row-50">
+                    <div class="col s12 center">
+                        <a id="choose-button" class="waves-effect waves-light btn-large disabled">choose</a>
+                    </div>
+                </div>
+                <div class="row row-100">
+                    <div class="col s12">
+                        <a class="btn-small right" href="index.html">Back to my menu</a>
+                    </div>
+                </div>
+            </section>
+        `;
+        super.renderHtml(htmlView);
+      }
+
+      setupProperties() {
+        this.languagePicker = $('#select-language');
+        this.chooseButton = $('#choose-button');
+        this.getAvailableLanguage();
+      }
+
+      setupEventListeners() {
+        this.languageValueChanged = this.languageValueChanged.bind(this);
+        this.chooseButtonClicked = this.chooseButtonClicked.bind(this);
+
+        this.languagePicker.change(this.languageValueChanged);
+        this.chooseButton.on('click', this.chooseButtonClicked);
+      }
+
+      presentNextController() {
+        const librarySelectMediaLanguageViewController = new LibrarySelectMediaLanguageViewController();
+        librarySelectMediaLanguageViewController.language = this.language;
+        librarySelectMediaLanguageViewController.shortLanguage = this.shortLanguage;
+        this.navigationController.present(librarySelectMediaLanguageViewController);
+      }
+
+      languageValueChanged(){
+        var e = document.getElementById("select-language");
+        this.language = e.options[e.selectedIndex].text;
+        this.shortLanguage = e.options[e.selectedIndex].value;
+        this.setupChooseButton();
+      }
+
+      setupChooseButton() {
+          const shouldEnableSelectButton = this.language !== "";
```

Comment:
Nebolo by citatelnejsie a vhodnejsie tu pouzit namiesto `this.language !== ""` nejake metodu na stringu `isEmpty` alebo nieco podobne?